var/spack/repos/builtin/packages/py-fenics-basix/package.py

```diff
@@ -35,6 +35,7 @@ class PyFenicsBasix(PythonPackage):
     depends_on("py-numpy@1.21:", type=("build", "run"))
     depends_on("py-pybind11@2.9.1:", when="@:0.7", type="build")
     depends_on("py-nanobind@1.5.1:", when="@0.8:", type="build")
+    depends_on("py-scikit-build-core+pyproject@0.5.0:", when="@0.8:", type="build")
```

Comment:
```suggestion
    depends_on("py-nanobind@1.6:", when="@0.8:", type="build")
    depends_on("py-scikit-build-core", when="@0.8:", type="build")
```
Based on `pyproject.toml`, unless I'm missing something.

Reply:
I'm not sure what you mean. Deleting `+pyproject` will cause it to fail. Plus we need a new enough version (0.5.0+)

Reply:
@adamjstewart Could you provide a little more clarity on the above? Thanks.

Reply:
I was just using the documented dependencies from main, you may want to update this in `pyproject.toml` if those are indeed required.

Reply:
@adamjstewart I've fixed the `pyproject.toml` file in Basix, adding `pyproject` as a build dependency, see PR https://github.com/FEniCS/basix/pull/741 and the line https://github.com/FEniCS/basix/blob/d11d807fab6341b9dcec57d8b89f201d9c6e8636/python/pyproject.toml#L2.

Reply:
Thanks, I updated my suggested changes to match the current state of your `pyproject.toml`. Feel free to update the versions either here or in `pyproject.toml`.

Reply:
`scikitb-build-core` version constraint added to updated `pyproject.toml` file, https://github.com/FEniCS/basix/blob/15b915bad133c4c77c9df5c6476506c4960365f5/python/pyproject.toml#L2.

Reply:
Now nanobind is the last remaining thing that needs to be updated

Reply:
Oops, overlooked that one. Fixed now.